ring-jetty-adapter/test/ring/adapter/test/jetty.clj

```diff
@@ -115,6 +120,44 @@
       (is (thrown-with-msg? ConnectException #"Connection refused"
                             (http/get test-url)))))
 
+  (testing "HTTPS server that needs client certs"
+    (with-server client-cert-handler {:client-auth :need
+                                      :keystore "test/keystore.jks"
+                                      :key-password "password"
+                                      :port test-port
+                                      :ssl? true
+                                      :ssl-port test-ssl-port}
+      (let [response (try (http/get test-ssl-url {:insecure? true})
+                          (catch Exception e :exception-caught))]
```

Comment:
`:throw-exceptions false` works perfectly for suppressing exceptions thrown by handler, but it seems unable to avoid exceptions during TLS handshake, which may have to be wrapped in a `try-catch` clause for this block.